internal/plan/plan.go

```diff
@@ -184,6 +187,76 @@ func (s *Service) Equal(other *Service) bool {
 	return reflect.DeepEqual(s, other)
 }
 
+// GetCommand overrides the default arguments (inside [ ]) if
+// CmdArgs is present and returns the command as a stream of strings.
+func (s *Service) GetCommand() ([]string, error) {
+	args, err := shlex.Split(s.Command)
+	if err != nil {
+		return nil, fmt.Errorf("cannot parse service %q command: %s", s.Name, err)
+	}
+	fargs := make([]string, 0)
+	for _, arg := range args {
+		if arg == "[" || arg == "]" {
+			if len(s.CmdArgs) > 0 {
+				break
+			} else {
+				continue
+			}
+		}
+		fargs = append(fargs, arg)
+	}
+	for _, arg := range s.CmdArgs {
+		fargs = append(fargs, arg)
```

Comment:
Note that you can do `fargs = append(fargs, s.CmdArgs...)` to append a slice all at once without the loop.

Reply:
Thanks!